app/full_text_search.py

```diff
@@ -0,0 +1,64 @@
+from const import PERSON_FULLTEXT_INDEX_NAME
+
+from google.appengine.api import search
+import re
+import model
+import logging
+
+def search_with_index(repo, query_obj, max_results):
```

Comment:
Add comment for each method using """ .... """ format. See:
https://google-styleguide.googlecode.com/svn/trunk/pyguide.html#Comments